packages/react/src/components/CardEditor/CardEditForm/_card-edit-form.scss

```diff
@@ -282,3 +282,7 @@ div.#{$iot-prefix}--card-edit-form--data-item-list {
     padding-right: $layout-05;
   }
 }
+
+.addAggregation {
+  margin-top: 1.2rem;
```

Comment:
use carbon guidelines https://carbondesignsystem.com/guidelines/spacing/overview/. BTW do we need this ?

Reply:
Yes we need this for new button

Reply:
+1 for token usage

Reply:
Fixed